lib/metrics/runtime.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const v8 = require('v8')
+const loopDelay = require('event-loop-delay')
```

Comment:
Shouldn't we add a conditional here checking if we're running v11.10.0 or newer, in which as we should use [the version built into core](https://nodejs.org/api/perf_hooks.html#perf_hooks_perf_hooks_monitoreventloopdelay_options) instead?

Reply:
The built-in version is somewhat different, so it wasn't clear to me how to use `event-loop-delay` as a fallback.